admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -0,0 +1,524 @@
+  name: my-rhdh
+  annotations:
+    # Below annotation is to specify if the loadbalancer is "internal" or "internet-facing"	   
+    alb.ingress.kubernetes.io/scheme: internet-facing
+
+    alb.ingress.kubernetes.io/target-type: ip
+
+    # TODO: Using an ALB HTTPS Listener requires a certificate for your own domain. Fill in the ARN of your certificate, e.g.:
+    # alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-west-2:xxxx:certificate/xxxxxx
+
+    # TODO: The HTTPS listener below requires setting the certificate ARN above. Remove it if you plan to expose your RHDH differently, for example via a CloudFront distribution.
+     alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS":443}]'
+
+
+      # TODO: if needed, set HTTP to HTTPS redirects. Every HTTP listener configured will be redirected to below mentioned port over HTTPS.
+      # alb.ingress.kubernetes.io/ssl-redirect: '443'
+
+spec:
+  # alb because we are using ALB.
+  # But adjust if using a different Ingress Controller
+  # and remove the 'alb.*' annotations accordingly.
+  ingressClassName: alb
+  rules:
+    - http:
+        paths:
+        - path: /
+          pathType: Prefix
+          backend:
+            service:
+              # TODO: my-rhdh is the name of your Backstage Custom Resource.
+              # Adjust if you changed it!
+              name: backstage-my-rhdh
+              port:
+                name: http-backend
+
+----
+
+[NOTE]
+====
+Configuring the HTTPS listener with Application Load Balancer (ALB) requires a certificate for your custom domain. In case, the custom domain or certificate is not available, then you can set the `alb.ingress.kubernetes.io/listen-ports` annotation to `[{"HTTP": 80}]` and create a CloudFront distribution using the ALB endpoint as the content origin. Using this approach, you can access the {product-short} using the CloudFront domain name. For more information about creating a CloudFront distribution, see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-creating.html[Steps for creating a distribution].
+====
+--
+
+. (Optional) If an ALB or another ingress controller is used to expose your {product-short} instance and you are unaware of the DNS hostname, then identify the provisioned DNS name and update the {product-short} configuration using the following steps:
+
+.. Run the following command to find the ingress hostname:
++
+--
+[source]
+----
+$ kubectl get ingress rhdh-developer-hub   
+
+NAME                    CLASS    HOSTS                                                                 ADDRESS                                                               PORTS   AGE
+rhdh-developer-hub   <none>   k8s-myns-rhdhde-3dec682266-491020386.eu-north-1.elb.amazonaws.com   k8s-myns-rhdhde-3dec682266-491020386.eu-north-1.elb.amazonaws.com   80      51m
+----
+--
+
+.. Create a ConfigMap named `app-config-rhdh` using the following template:
```

Comment:
ConfigMap creation with the required information (like the domain name) is now done in the very first steps, before creating the Custom Resource.

Reply:
Done